### DISCUSSION TITLE
3.15版本PDF无法预览，onlyoffice无法预览

### DISCUSSION THREAD

**Discussion by Masterlovsky** [AlistGo/alist#3970] [General]
![Snipaste_2023-03-27_09-22-13](https://user-images.githubusercontent.com/45965357/227819748-f16549ff-5898-4a60-bc39-910027a40d5b.png)

**Comment by Rektyfikowany** [comment]
无法复现，有复现链接或者文件吗

**Reply by Masterlovsky** [reply]
我是内网环境下docker安装的alist-aria2，这边儿的测试结果是onlyoffice的文件会报一个401错误代码，PDF如上图，现在换回14.0版本之后就一切正常了。应该和文件无关，所有的PDF打开都报这个错

**Reply by Rektyfikowany** [reply]
检查一下签名设置，有没有#3957 同样的问题，如有请使用修复的版本[runs/4523610315](https://github.com/alist-org/alist/actions/runs/4523610315)

**Reply by chenjuefei** [reply]
存储》编辑》启用签名

**Reply by Winyer** [reply]
> 存储》编辑》启用签名

请问启用签名的作用是？

**Reply by Winyer** [reply]
> 存储》编辑》启用签名

还有签名所有这个功能...文档也没找到说明
